download/releaseNotes/releaseNotes8.adoc

```diff
@@ -1,17 +1,95 @@
 = Release Notes 8
-:awestruct-description: New and noteworthy, demos and status for OptaPlanner 7.
+:awestruct-description: New and noteworthy, demos and status for OptaPlanner 8.
 :awestruct-layout: releaseNotesBase
 :awestruct-priority: 1.0
 :awestruct-release_notes_version: 8
-:awestruct-release_notes_version_qualifier: Final
+:awestruct-release_notes_version_qualifier: Beta
 
-== New and noteworthy: Engine 8.0.0.Final
+== What is OptaPlanner 8?
 
-=== XML Schema Definition for both solverConfig.xml and benchmarkConfig.xml
+*OptaPlanner 8 is basically OptaPlanner 7 without the deprecated methods*
+and a few minor backwards incompatible changes.
+**Migrating your code from 7 to 8 is easy and quick.**
+Just follow the instructions in link:../upgradeRecipe/upgradeRecipe8.html[the upgrade recipe].
 
-In this release, we are introducing XML Schema Definition for an XML configuration of both the solver and the benchmark.
-IDEs can now offer code completion of the solver and the benchmark configuration.
+As usual, Red Hat offers link:../../product/services.html[enterprise support and consulting] on OptaPlanner 8,
+with the _Red Hat build of OptaPlanner_ in an upcoming release of Red Hat Decision Manager 7
+in the first half of 2021.
```

Comment:
As far as I know, Red Hat does not make product release commitments. Even less so when it's on a community page somewhere. Please remove or, at the very least notify PM, at which point they will say the same thing. :-)

Reply:
My understanding was that we can make such promises only inside the company. I would definitely not mention any product release plans publicly.

Reply:
I've checked this with PM, they approve and gave me this messaging, including the timeframing.
It's important for Red Hat customers and especially potential customers to know we have their back: they can start now with RHDM's OptaPlanner 7 and upgrade to 8 soon. That's a critical part of the OptaPlanner 8 messaging.